core/ecschema-rpc/impl/README.md

```diff
@@ -0,0 +1,9 @@
+﻿# iModel.js RPC Interface Integration Tests
+
+Copyright © Bentley Systems, Incorporated. All rights reserved.
+
+This package contains the Schema RPC Interface Implementation which can be added to a backend.
+
+## Testing
+
+The tests for this RPC Interface are available in the [@bentley/ecschema-rpcinterface-tests](../../../full-stack-tests/ecschema-rpc-interface) package, please visit the README in that package for more information.
```

Comment:
The link here will want to be to the package on npmjs, so something similar to `https://www.npmjs.com/package/@bentley/rpcinterface-full-stack-tests`